<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ugwindow</class>
 <widget class="QDialog" name="ugwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0.0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(50, 50, 50, 255));</string>
  </property>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>292</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Zur√ºck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightVorraum">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>148</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Vorraum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightTechnik">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>148</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Technik</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightLager">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>148</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Lager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightFitness">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Fitness</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightArbeit">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>4</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Arbeit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightStiege">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>4</y>
     <width>195</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Stiege</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
